src/index.html

```diff
@@ -13,20 +14,32 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./styles/style.css"
```

Comment:
@jessikamorais, you should not change the path of the styles.css file. The PR verification has pre-files selected to verify whether the code you are doing is correct or not. As you created a new file styled under the/styles folder, the verification is not working. Please move this code back to `href="style.css"` and move your whole CSS code inside that file to make code verification work properly.